loco-new/base_template/src/tasks/user_delete.rs

```diff
@@ -0,0 +1,59 @@
+use loco_rs::prelude::*;
+use dialoguer::{theme::ColorfulTheme, Confirm};
```

Comment:
Do new Cargo projects include `dialoguer` by default?
If not, I’d prefer not to add it to the project (io::stdout).